roles/util-cfg-service/templates/info-typer.properties.j2

```diff
@@ -16,13 +16,12 @@ info-typer.irods.retry-sleep = 1000
 info-typer.irods.use-trash   = true
 
 # AMQP settings
-info-typer.amqp.host                 = {{ irods.amqp_broker.host }}
-info-typer.amqp.port                 = {{ irods.amqp_broker.port }}
-info-typer.amqp.user                 = {{ irods.amqp_broker.user }}
-info-typer.amqp.pass                 = {{ irods.amqp_broker.password }}
+info-typer.amqp.uri                  = amqp://{{ irods.amqp_broker.user }}:{{ irods.amqp_broker.password }}@{{ irods.amqp_broker.host }}:{{ irods.amqp_broker.port }}/%2F
```

Comment:
FYI. When the iRODs AMQP exchange is migrated, this property will be reduced to `{{ amqp_irods_uri }}`. 
For now, this is perfect :+1: